app/views/hub/users/edit_role.html.erb

```diff
@@ -61,7 +61,8 @@
         <div class="form-group">
           <%= label_tag(:sites, t("devise.invitations.new.site_label"), class: "form-question") %>
           <div class="select form-width--long">
-            <%= hidden_field_tag(:sites, (@role&.respond_to?(:sites) ? @role.sites : []).map { |site| { value: site.id, name: site.name }}.to_json.to_s.html_safe, class: "multi-select-vita-partner select__element") %>
+            <% site_value = taggable_format((@role&.respond_to?(:sites) ? @role.sites : []).map { |site| { value: site.id, name: site.name }}) %>
```

Comment:
This is a good example of always returning the same data type no matter which side of the ternary executes.